app/db/models/user.js

```diff
@@ -42,13 +42,13 @@ module.exports = (Sequelize, db) => {
     },
 
     // account type
-    //   RESTRICTED - not used currently
+    //   RESTRICTED - not used currently / email not verified yet
```

Comment:
now it's used so you can replace "not used currently" with "an unverified user/member" or something